util/make_timeline.py

```diff
@@ -97,14 +139,29 @@ def parse_file(args):
 
     entries = []
     started = False
-
+    encounter_sets = []
     with args.file as file:
+        # If searching for encounters, divert and find start/end first3
+        if args.search_fights:
+            encounter_sets = find_fights_in_file(file)
+            # If all we want to do is list encounters, stop here and give to the user.
+            if args.search_fights < 0:
+                return [f'{i + 1}. {e_info[0]} {e_info[1]} {e_info[2]}' for i, e_info in enumerate(encounter_sets)]
```

Comment:
Can you leave a comment somewhere (here or maybe line 156) that fight indices start at one?

Reply:
Perhaps something to the effect of "This is to make 1-based indices transparent to the user"? I don't like the hardcoded offsets, but I'm not sure whether there's a better solution that's a good user experience.